### PR TITLE
Salt Day 10 part 3

### PR DESCRIPTION
**Pull request by Parzivus**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_7B.txt [hunk 1]
```diff
@@ -12,7 +12,8 @@ I can't spot Arcueid anywhere.
 [sha:71dfe432f9690fbdea03c738861b76a28ad5f1d4]{
 -- Page 10022, Offset 26340.
 -- 街は広すぎて、何の手がかりもなしで人間ひとりを捜し当てるのには時間が足りない。
-The city is too big, and I have no clues to go off.
+// Sounds better to me, curious what others think
+The city is too big, and I have no clues to go off of.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_7B.txt:16]: Ngrams suggests that the most common use is actually clues to go _on_

File: script/Arcueid/Day 10/10_00_ARC10_7_2B.txt [hunk 2]
```diff
@@ -1160,7 +1162,8 @@ Rather than violating her on the spot, a struggle to the death is far more appro
 [sha:4ac4d24a913677a249b3a8448971a11cce087d60]{
 -- Page 829, Offset 26609.
 -- 片腕が女の白い、白い白い白い美しい喉を掴む。
-With one arm, I grab onto her beautiful, white white white white throat.
+// Thoughts?
+With one arm, I grab onto her beautiful, white, white, white, white throat.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_7_2B.txt:1166]: It's a little less insane but no strong feeling

File: script/Arcueid/Day 11/11_00_ARC11_3.txt [hunk 3]
```diff
@@ -887,7 +887,7 @@ I mean, isn't it lonely to always be out there on your own?"
 [sha:7ef6dc156f74976249afea1d52ca7a5bb0c79955]{
 -- Page 212, Offset 30116.
 -- アルクェイドは答えない。
-Arcueid does not respond.
+She does not respond.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 11/11_00_ARC11_3.txt:890]: Per prior comment I'd undo this and later changes

File: script/Arcueid/Day 10/10_00_ARC10_8_2.txt [hunk 4]
```diff
@@ -107,7 +107,8 @@ This is not something that can be forgiven with a mere 'I'm sorry'.
 [sha:a6ecfe79db3bdfb8e6c5ad59e93724354872683c]{
 -- Page 870, Offset 26791.
 -- 「気にしないで。……謝らなくちゃいけないのは、わたしの方なんだから」
-"Don't worry about it... I'm the one who should be apologizing"
+// This should have a period despite the JP text, I think
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_8_2.txt:110]: Yeah I think speech should be properly punctuated in EN even when it isn't in the JP. Just easy to miss.

File: script/Arcueid/Day 10/10_00_ARC10_6.txt [hunk 5]
```diff
@@ -385,8 +386,9 @@ Right now I'm still clueless about what kind of vampire Arcueid's enemy is.
 [sha:38edf340f1b2d98672db4132c98193ff44201fa5]{
 -- Page 699, Offset 26147.
 -- 「……俺は助かるけど。いいのか、そんな話をして」
-"...That would really help me out.
- But is it okay for you to tell me?"
+// Should either do it like this or remove the "but" IMO
+"...That would really help me out,
+ but is it okay for you to tell me?"
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_6.txt:391]: Maybe rearrange to 
> help me out. Are you sure it's okay for you to tell me, though?

File: script/Arcueid/Day 10/10_00_ARC10_5E.txt [hunk 6]
```diff
@@ -710,7 +713,7 @@ Not drinking blood even once sounds so sweet that it makes me wanna cry, Tohno.
 -- 「アルクェイド……？」
 "Arcueid...?"
 }
-[offset:26065]{
+[sha:7ef6dc156f74976249afea1d52ca7a5bb0c79955]{
```
[the remark this answers is not in the document]
Comment (Parzivus) [reply script/Arcueid/Day 10/10_00_ARC10_5E.txt:716]: How do I get it to pass the lint check without homogenizing it?

[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_5E.txt:715]: Put the override back, same as it was before

**Pull request merged**